tests/DependencyInjection/FixtureMonologExtensionTestCase.php

```diff
@@ -191,9 +192,11 @@ public function testPsr3MessageProcessingDisabledOnNullHandler()
 
     public function testHandlersV2()
     {
-        $this->getContainer('handlers');
+        $container = $this->getContainer('handlers');
 
-        $this->expectNotToPerformAssertions();
```

Comment:
I improved this test because I initially added the `expectUserDeprecationMessage` to `XmlMonologExtensionTest::loadFixture`. But Symfony 7.3 does not trigger this deprecation and `#[IgnoreDeprecation]` is better.
Keeping the test update as it still covers the configuration.

Reply:
shouldn't this test improvement be done in the 3.x branch already and merged up ?

Reply:
Yes: https://github.com/symfony/monolog-bundle/pull/544